app/controllers/administrateurs/procedures_controller.rb

```diff
@@ -417,13 +416,14 @@ def filter_procedures(filter)
       procedures_result = procedures_result.where(hidden_at_as_template: nil)
       procedures_result = procedures_result.where(aasm_state: filter.statuses) if filter.statuses.present?
       procedures_result = procedures_result.where("tags @> ARRAY[?]::text[]", filter.tags) if filter.tags.present?
+      procedures_result = procedures_result.where(is_template: filter.is_template) if filter.is_template.present?
```

Comment:
avec les refactos dont je parle plus bas ça donnera : 

```suggestion
      procedures_result = procedures_result.where(template: true) if filter.template?
```